### PR TITLE
✨ PIC-1121 Add probationStatus object to search and match response

### PR DESCRIPTION
**Pull request by chris-faulkner-mojdt**

### CODE REVIEW

**Review by andymarke — DISMISSED** [review]
LGTM

File: src/test/resources/elasticsearchdata/john-smith.json [hunk 1]
```diff
@@ -78,5 +78,11 @@
     "officer": {"code": "NO2AAMU", "forenames": "Unallocated", "surname": "Staff"},
     "probationArea": {"code": "NO2", "description": "NPS London"},
     "notes": "Level 1 Cat 2 notes"
+  },
+  "probationStatus": {
```
Comment (mikehalmamoj) [inline src/test/resources/elasticsearchdata/john-smith.json:82]: I was just looking at the ConvictionService in community-api and it looks like you can get null values for termination date and in breach... I wonder if we could do with some tests around those null values?  (In another PR if you think it is necessary)

**Review by mikehalmamoj — APPROVED** [review]

**Review by jhevans — DISMISSED** [review]
Looks good.  Mike's comment about checking the nullable fields is a good call though so would be good to get that done

File: src/test/resources/elasticsearchdata/john-smith.json (hunk 1 above)
Comment (chris-faulkner-mojdt) [reply src/test/resources/elasticsearchdata/john-smith.json:82]: Fair point.

**Review by jhevans — APPROVED** [review]

**Pull request merged**